pkg/storage/chunk/client/aws/s3_storage_client.go

```diff
@@ -85,8 +69,8 @@ type S3Config struct {
 	BucketNames      string
 	Endpoint         string              `yaml:"endpoint"`
 	Region           string              `yaml:"region"`
-	AccessKeyID      secret              `yaml:"access_key_id"`
-	SecretAccessKey  secret              `yaml:"secret_access_key"`
+	AccessKeyID      flagext.Secret      `yaml:"access_key_id"`
```

Comment:
I would like us to stay consistent with Mimir and Tempo and not consider AccessKeyID as a secret.

https://github.com/grafana/mimir/blob/c248b4e81e8fed041b188749719bb8167aa77a1f/pkg/storage/bucket/s3/config.go#L78

https://github.com/grafana/tempo/blob/4e0f12da3cd2d665e5933bb402b06ca66efe5d7a/tempodb/backend/s3/config.go#L13

(It is actually called `access_key` there :sob:)